internal/http/services/owncloud/ocs/handlers/cloud/users/users.go

```diff
@@ -65,6 +66,7 @@ type Users struct {
 	Quota       *Quota `json:"quota" xml:"quota"`
 	Email       string `json:"email" xml:"email"`
 	DisplayName string `json:"displayname" xml:"displayname"`
+	UserType    string `json:"usertype" xml:"usertype"`
```

Comment:
You added `user-type` to the other struct. I would keep the format consistent, so either add the `-` here or remove it from the other struct.

Reply:
That's because it's the same for `display-name`, so I thought it's by design. Should I fix both? That'll probably break the tests though

Reply:
Ah I see. Yeah, don't fix the display name in here. We had some problems with that before so I think that should be tackled in another PR. The new field though should be introduced consistently so I would vote for `user-type`.

Reply:
Cool, will do